client/app/common/less/link.less

```diff
@@ -0,0 +1,6 @@
+.oui-link {
```

Comment:
Should be in ui-kit ?

Reply:
Yes, this should actually be in ui-kit. We have a seperate bug to track this issue in UI-kit. Until then, this code is a work-around,

Reply:
This is done :)